src/app/(proper_react)/(redesign)/(public)/how-it-works/HowItWorksView.test.tsx

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { it, expect } from "@jest/globals";
+import { composeStory } from "@storybook/react";
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import Meta, { HowItWorks } from "./HowItWorksView.stories";
+import { userEvent } from "@testing-library/user-event";
+import { useSession } from "next-auth/react";
+import { deleteAllCookies } from "../../../../functions/client/deleteAllCookies";
+import { useTelemetry } from "../../../../hooks/useTelemetry";
+
+jest.mock("next-auth/react", () => {
+  return {
+    signIn: jest.fn(),
+    useSession: jest.fn(() => {
+      return {};
+    }),
+  };
+});
+
+jest.mock("../../../../hooks/useTelemetry");
+
+beforeEach(() => {
+  // For reasons that are unclear to me, the mock implementation defind in the
+  // call to `jest.mock` above forgets the implementation. I've spent way too
+  // long debugging that already, so I'm settling for this :(
+  const mockedUseSession = useSession as jest.Mock;
+  mockedUseSession.mockReturnValue({});
+
+  // Make the rebrand announcement banner show up by default
+  deleteAllCookies();
+});
+
+describe("How it works page", () => {
+  it("passes the axe accessibility test suite", async () => {
+    const ComposedPage = composeStory(HowItWorks, Meta);
+    const { container } = render(<ComposedPage />);
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("Data Removal buttons enter user into premium subscription flow", () => {
+    const ComposedPage = composeStory(HowItWorks, Meta);
+    render(<ComposedPage />);
+    const dataRemovalBtns = screen.getAllByRole("link", {
+      name: "Get data removal",
+    });
+    const dataRemovalBtn1 = dataRemovalBtns[0];
+    const dataRemovalBtn2 = dataRemovalBtns[1];
+    expect(dataRemovalBtn1).toHaveAttribute(
+      "href",
+      "https://accounts.firefox.com/subscriptions/products/prod_OiV9RSaatywSRy?plan=price_1Nv4ODJNcmPzuWtRoYpoFHXd",
+    );
+    expect(dataRemovalBtn2).toHaveAttribute(
+      "href",
+      "https://accounts.firefox.com/subscriptions/products/prod_OiV9RSaatywSRy?plan=price_1Nv4ODJNcmPzuWtRoYpoFHXd",
+    );
+  });
+
+  it("Free Scan button enters user into account.mozilla.com", async () => {
+    const mockedRecord = useTelemetry();
+    const ComposedPage = composeStory(HowItWorks, Meta);
+    render(<ComposedPage />);
+
+    const user = userEvent.setup();
+
+    const getFreeScanBtn1 = screen.getAllByRole("button", {
+      name: "Get free scan",
+    })[0];
+    const getFreeScanBtn2 = screen.getAllByRole("button", {
+      name: "Get free scan",
+    })[1];
```

Comment:
Not sure if we want to cache the selector like we did above on L47-51:
```js
    const dataRemovalBtns = screen.getAllByRole("link", {
      name: "Get data removal",
    });
    const dataRemovalBtn1 = dataRemovalBtns[0];
    const dataRemovalBtn2 = dataRemovalBtns[1];
```

We might be able to almost one-line this using destructuring:
```js
    const [getFreeScanBtn1, getFreeScanBtn2] = screen.getAllByRole("button", {
      name: "Get free scan",
    });
```

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4632/commits/d3fdb2c46e1125123194a1f20dd3e13512aad5e5